addons/hr_holidays_attendance/models/hr_leave_allocation.py

```diff
@@ -23,7 +23,7 @@ def default_get(self, fields):
 
     overtime_deductible = fields.Boolean(compute='_compute_overtime_deductible')
     overtime_id = fields.Many2one('hr.attendance.overtime', string='Extra Hours', groups='hr_holidays.group_hr_holidays_user')
-    employee_overtime = fields.Float(related='employee_id.total_overtime')
+    employee_overtime = fields.Float(related='employee_id.total_overtime', store=True)
```

Comment:
I'm afraid this cannot be done in a stable release as it implies a change of schema :-(

Reply:
Indeed... Good catch, thanks! I have removed the store=True and solved the conditions in _compute_from_holiday_type :-)